test/browser/spider.js

```diff
@@ -27,13 +60,26 @@ function crawl() {
       resourceData,
       queueItem
     ]);
+    var unicodeResourceData = resourceData.toString("utf8");
 
-    resources.push.apply(resources, findSrcSetURLs(
-      resourceData.toString("utf8"),
-      queueItem.url
-    ));
+    URL_FINDERS.forEach(function(findURLs) {
+      resources.push.apply(resources, findURLs(
+        unicodeResourceData,
+        queueItem.url
+      ));
```

Comment:
Ah nvm, I get it now :grin:  
We are calling `apply` so we can use `push` in a way similar to how we use `concat`, since we can just pass an array as an argument instead of looping through that array and call `resources.push` multiple times